docs/api-conventions.md

```diff
@@ -55,15 +80,17 @@ Labels are intended for organizational purposes by end users (select the pods th
 
 #### Spec and Status
 
-By convention, the Kubernetes API makes a distinction between the specification of the desired state of a resource (a nested object field called "spec") and the status of the resource at the current time (a nested object field called "status"). The specification is persisted in stable storage with the API object and reflects user input. The status is generated at runtime and summarizes the current effect that the spec has on the system. 
+By convention, the Kubernetes API makes a distinction between the specification of the desired state of an object (a nested object field called "spec") and the status of the object at the current time (a nested object field called "status"). The specification is persisted in stable storage with the API object and reflects user input. The status is summarizes the current state of the object in the system, and is usually persisted with the object by an automated processes (but may be created on the fly).
```

Comment:
One thing I would like to add re. status, since it has come up a couple times recently: Even if persisted, status MUST be reconstructable from other sources.